tasks/managed/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -36,6 +36,10 @@ spec:
     - name: taskGitRevision
       type: string
       description: The revision in the taskGitUrl repo to be used
+    - name: internalRequestServiceAccountName
```

Comment:
In sign-index-image and rh-sign-image we have this:
```
          service_account_name=$(jq -r '.spec.pipeline.serviceAccountName // "appstudio-pipeline"' "${RPA_FILE}")
```

I wonder what's the best way to handle it here. Have you considered using the same? But I guess the issue is that for internal clusters internal-pipeline will be used for signing and this `release-service-account` will only be available in the app sre cluster, right?

It just seems inconsistent to use a task param here, but maybe it's the best way.

Reply:
I did not, but your suggestion actually make me think that the service account would better be set in the data,  with the default to `release-service-account`.

About the account, that is correct. As the service account already did exist in the appsre clusters (appstudio-pipeline does not) and has all the necessary rbac set on it, we would use it for the internalrequest. 

Also the best as it seems, would be adjust the appsre cluster to have the appstudio-pipeline account as well, making it even more consistent. I will create a ticket to adjust this in the near future, as we need to deliver this still this sprint.

Reply:
ok, thanks. I agree.

> appstudio-pipeline does not

Oh, I didn't notice that this was only set for the use case of internal-pipelinerun. OK.